src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -32,6 +36,8 @@ const DragDropcontext: FC<dragDropProps> = ({
   const [taskList, setTaskList] = useState<Array<task>>(unAssignedTasks);
   const [memberList, setMemberList] = useState<Array<string>>(idleMembers);
   const [isTaskOnDrag, setIsTaskOnDrag] = useState<boolean>(false);
+  const [draggableId1, setDraggableId1] = useState<string>('');
+  const [draggableId2, setDraggableId2] = useState<string>('');
```

Comment:
what are these draggableId1 and 2? can you give more context so we can come up with better variable names

Reply:
draggableId1, draggableId2 are ids of task and member whenever we drop any task on member or vice-versa.

Reply:
Let's call them `taskId` and `memberId` in that case?

Reply:
Actually we can't because we can't say which one is taskId and memberId since, we are dragging task over member or member over task.

Reply:
I see, makes sense